docker-compose.yaml

```diff
@@ -4,38 +4,9 @@ services:
     build: .
     image: ffc-demo-user-service
     container_name: ffc-demo-user-service
-    command: sh -c "scripts/wait-for/wait-for ffc-demo-user-postgres:5432 -- npm run migrate && npm run start:watch"
-    depends_on:
-      - ffc-demo-user-postgres
+    command: sh -c "npm run start:watch"
     environment:
+      NODE_ENV: production
       POSTGRES_DB: ffc_demo_users
-      POSTGRES_PASSWORD: ppp
-      POSTGRES_USERNAME: postgres
-      POSTGRES_HOST: ffc-demo-user-postgres
       POSTGRES_PORT: 5432
-    networks:
-      - ffc-demo-users
-    ports:
-      - "3002:3002"
-    volumes:
-      - node_modules:/usr/src/app/node_modules
-      - .:/usr/src/app

```

Comment:
The ports will need to move out of the main docker compose due to the way Jenkins run all tests on the same server - we could end up with two PRs on the same repo failing due to both trying to bind to the same port.

It's fine to leave as it is for now, but needs to change in future. This can be done along with other code changes to ensure tests run in isolation in Jenkins.